src/services/commandService.ts

```diff
@@ -1,3 +1,23 @@
-export function parse(rawCommand: string): {} {
-    return { raw: rawCommand }
+export function parseGive(rawCommand: string): {} {
```

Comment:
Instead of returning `{}` please create an interface for returned object e.g. 
`interface KudosTransactionInfo {
  kudosReceiverId: string,
 kudosTransactionValue: number
 }`
That will help in future development